lib/static/components/controls/gui-controls.js

```diff
@@ -33,27 +34,32 @@ class GuiControls extends Component {
     }
 
     render() {
-        const {actions, suiteIds, failed, running, autoRun, processing} = this.props;
+        const {actions, suiteIds, failed, running, serverStopped, autoRun, processing} = this.props;
 
         return (
             <Fragment>
                 <div className="control-buttons">
                     <RunButton
                         autoRun={autoRun}
-                        isDisabled={!suiteIds.all.length || processing}
+                        isDisabled={!suiteIds.all.length || processing || serverStopped}
                         handler={actions.runAllTests}
                         isRunning={running}
                     />
                     <ControlButton
                         label="Retry failed tests"
-                        isDisabled={!failed.length || processing}
+                        isDisabled={!failed.length || processing || serverStopped}
                         handler={this._runFailedTests}
                     />
                     <ControlButton
                         label="Accept opened"
-                        isDisabled={!failed.length || processing}
+                        isDisabled={!failed.length || processing || serverStopped}
```

Comment:
I suggest to refactor this piece of code. Let's create a function for this in which we get length of the first argument and add `|| processing || serverStopped`